modules/power-monitoring-deleting-power-monitor-crd.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/uninstalling-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-deleting-power-monitoring-crd_{context}"]
+= Deleting the PowerMonitor custom resource definition
+
+You can delete the `PowerMonitor` custom resource definition (CRD) by removing the `power-monitor` instance of the `PowerMonitor` CRD from the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the {product-title} web console.
+* You are logged in as a user with the `cluster-admin` role.
+
+.Procedure
+
+. In the web console, go to *Operators* → *Installed Operators*.
+
+. Click *{PM-title-c}* from the *Installed Operators* list and go to the *{PM-kepler}* tab.
```

Comment:
```suggestion
. Click *{PM-title-c}* from the *Installed Operators* list and go to the *{PM-powermonitor}* tab.
```

<img width="788" height="98" alt="Image" src="https://github.com/user-attachments/assets/c8a252f2-081e-4d80-b1a1-59a3359f0af5" />

In that screenshot `Kepler` tab is incorrect. 

More context: 

With TP 0.5 there 2 CRDs 
1. Kepler (old which has tab)
2. Power Monitor - new CRD  which has a new Tab